llm_api_hub/server/app.py

```
@@ -364,10 +409,60 @@ def models(request: Request) -> Dict[str, Any]:
 @app.post("/v1/chat/completions")
 def chat_completions(request: ChatCompletionRequest, raw_request: Request) -> Any:
     validate_auth(raw_request)
+    return handle_completion(request)
+
+@app.post("/v1/messages")
+def anthropic_messages(request: ChatCompletionRequest, raw_request: Request) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `anthropic_messages`

(ANN401)